app/controllers/actions/IdentifierAction.scala

```diff
@@ -92,7 +92,7 @@ class AuthenticatedIdentifierAction @Inject() (
         } yield newGroupEnrolment || legacyGroupEnrolment
 
         hasGroupEnrolment flatMap {
-          case true  => renderer.render("unauthorisedWithGroupAccess.njk").map(Unauthorized(_))
+          case true  => renderer.render("unauthorisedWithGroupAccessView").map(Unauthorized(_))
```

Comment:
This should be something like
          case true  => Redirect(controllers.routes.UnauthorisedWithGroupAccessController.onPageLoad())

Reply:
I think Matt is working on this page